larray_editor/utils.py

```diff
@@ -54,29 +57,25 @@ def get_package_version(package_name):
         return 'N/A'
 
 
-def get_versions():
-    """Get version information for components used by the Editor"""
+def get_versions(package='editor'):
```

Comment:
for an internal function, making package a required argument seems like a good idea (it is more explicit)